source/views/guides/update-guide.markdown

```diff
@@ -0,0 +1,197 @@
+# Diesel Update Guide
+
+This guide is intended for those who are somewhat familiar with Rust and have gone through the [basic diesel guide](http://diesel.rs/guides/getting-started/).
+
+This guide will walk you through more examples of updating records using Diesel and will build on top of the original tutorial.
+
+## Updating from the Diesel guide
+
+If you've followed the Diesel getting started guide, you'll have something that looks like this for updating a post:
+
+```rust
+/// src/bin/publish_post.rs
+extern crate diesel_demo;
+extern crate diesel;
+
+use self::diesel::prelude::*;
+use self::diesel_demo::*;
+use self::diesel_demo::models::Post;
+use std::env::args;
+
+fn main() {
+    use diesel_demo::schema::posts::dsl::{posts, published};
+
+    let id = args().nth(1).expect("publish_post requires a post id")
+        .parse::<i32>().expect("Invalid ID");
+    let connection = establish_connection();
+
+    let post = diesel::update(posts.find(id))
+        .set(published.eq(true))
+        .get_result::<Post>(&connection)
+        .expect(&format!("Unable to find post {}", id));
+    println!("Published post {}", post.title);
+}
+```
+
+First, we find the post by the id that we pass into the CLI. Then we call .set on the result where we set the column published to be true. Then we grab the result and pass that to our print line. 
+
+The next example will take you through updating other types of columns besides a bool. 
+
+## Passing multiple values into set
+
+Let's say instead of just setting published to true on your post, you also wanted to update the title to tell the world you've published this post. 
+
+```rust
+fn main() {
+    use diesel_update::schema::posts::dsl::{posts, published, title};
+
+    let id = args().nth(1).expect("publish_post requires a post id")
+        .parse::<i32>().expect("Invalid ID");
+    let connection = establish_connection();
+
+    let post = diesel::update(posts.find(id))
+        .set((published.eq(true), title.eq("I've been updated")))
```

Comment:
If we're telling the world we've published the post, should we update the title to `I've been published!`, or something similar?